Traceback (most recent call last):
  File "run.py", line 128, in <module>
    trade_action(get_position_info(), get_current_trend(), get_minute_candle())
  File "run.py", line 78, in get_position_info
    positionAmt = float(client.futures_position_information(symbol=get_symbol(), timestamp=get_timestamp())[0].get('positionAmt'))
  File "/home/zhenyean/.local/lib/python3.8/site-packages/binance/client.py", line 3688, in futures_position_information
    return self._request_futures_api('get', 'positionRisk', True, data=params)
  File "/home/zhenyean/.local/lib/python3.8/site-packages/binance/client.py", line 222, in _request_futures_api
    return self._request(method, uri, signed, True, **kwargs)
  File "/home/zhenyean/.local/lib/python3.8/site-packages/binance/client.py", line 197, in _request
    return self._handle_response()
  File "/home/zhenyean/.local/lib/python3.8/site-packages/binance/client.py", line 230, in _handle_response
    raise BinanceAPIException(self.response)
binance.exceptions.BinanceAPIException: APIError(code=-1021): Timestamp for this request is outside of the recvWindow.